src/realm/table.cpp

```diff
@@ -2719,17 +2719,6 @@ bool Table::compare_objects(const Table& t) const
 }
 
 
-void Table::check_lists_are_empty(size_t) const
```

Comment:
Made redundant by core 6's change to setting the pk as part of object creation rather than a set afterwards.